deployment/existing/auth_oidc/dex.tmpl

```diff
@@ -80,8 +80,13 @@ data:
       # After authenticating and giving consent, dex will redirect to
       # this url for the specific client.
       - {{.KubeflowEndpoint}}/login/oidc
+      {{if .OIDCExtraRedirectUris}}
```

Comment:
Instead of having `{{.KubeflowEndpoint}}/login/oidc` and `OIDCExtraRedirectUris` can this be made as `OIDCRedirectUris` instead?
cc: @yanniszark

Reply:
On this note https://github.com/kubeflow/kubeflow/pull/3607#discussion_r300729280

Reply:
I think this is best discussed with @yanniszark as he and I discussed previously

Reply:
Sure. We can resolve this separately.